pallets/proposals/src/benchmarking.rs

```diff
@@ -1,313 +1,366 @@
+        
+        let bounded_desc_f: BoundedDescriptionField = "b".repeat(<MaxDescriptionField as Get<u32>>::get() as usize).as_bytes().to_vec().try_into().unwrap();
+        
+        let milestones: BoundedProposedMilestones = vec![ProposedMilestone {
+            name: bounded_str_f.clone(),
+            percentage_to_unlock: 1,
+        }; 100].try_into().unwrap();
+
+        let required_funds: BalanceOf<T> = u32::MAX.into();
         let currency_id = CurrencyId::Native;
 
-    }: _(RawOrigin::Signed(caller.clone()), project_name.clone(), project_logo, project_description, website, milestones, required_funds, currency_id)
+    }: _(RawOrigin::Signed(whitelisted_caller()), bounded_str_f.clone(), bounded_str_f.clone(), bounded_desc_f.clone(), bounded_desc_f, milestones, required_funds, CurrencyId::Native)
     verify {
-        assert_last_event::<T>(Event::ProjectCreated(caller,project_name.clone(),0, required_funds, currency_id).into());
+        assert_last_event::<T>(Event::ProjectCreated(caller, bounded_str_f.to_vec(), 0, required_funds, CurrencyId::Native).into());
+    }
+
+    add_project_whitelist {
+        let caller = create_project_common::<T>(u32::MAX.into());
+        let mut bbt : BoundedWhitelistSpots<T> = BTreeMap::new().try_into().unwrap();
+
+        for i in 0..<MaxWhitelistPerProject as Get<u32>>::get() {
+            bbt.try_insert(whitelisted_caller(), 100u32.into()).unwrap();
+        }
+
+    }: _(RawOrigin::Signed(caller), 0, bbt)
+    verify {
+        assert_last_event::<T>(Event::WhitelistAdded(0, 1u32.into()).into());
+    }
+
+    remove_project_whitelist {
+        let caller = create_project_common::<T>(u32::MAX.into());
+        let mut bbt : BoundedWhitelistSpots<T> = BTreeMap::new().try_into().unwrap();
+        
+        for i in 0..<MaxWhitelistPerProject as Get<u32>>::get() {
+            bbt.try_insert(whitelisted_caller(), 100u32.into()).unwrap();
+        }
+        let _ = Proposals::<T>::add_project_whitelist(RawOrigin::Signed(caller.clone()).into(), 0, bbt);
+
+    }: _(RawOrigin::Signed(caller), 0)
+    verify {
+        assert_last_event::<T>(Event::WhitelistRemoved(0, 1u32.into()).into());
     }
 
     schedule_round {
-        create_project_common::<T>(CONTRIBUTION);
-        let start_block: T::BlockNumber = 0u32.into();
-        let end_block: T::BlockNumber = 10u32.into();
-        let project_keys: Vec<ProjectKey> = vec![0];
+        let mut project_keys: BoundedProjectKeys = vec![].try_into().unwrap();
+
+        for i in 0..<MaxProjectKeysPerRound as Get<u32>>::get() {
+            let _caller = create_project_common::<T>(u32::MAX.into());
+            let _ = project_keys.try_push(i).unwrap();
+        }
 
-    }: _(RawOrigin::Root, start_block, end_block, project_keys.clone(), RoundType::ContributionRound)
+    }: _(RawOrigin::Root, 1u32.into(), 100u32.into(), project_keys.clone(), RoundType::ContributionRound)
```

Comment:
I prefer variables rather than hard coded values because it makes it easier for the reader